lib/inspec-vault/input.rb

```diff
@@ -9,19 +9,25 @@ class Input < Inspec.plugin(2, :input)
     attr_reader :mount_point
     attr_reader :path_prefix
     attr_reader :vault
+    attr_reader :priority
 
     def initialize
       @plugin_conf = Inspec::Config.cached.fetch_plugin_config("inspec-vault")
 
       @mount_point = fetch_plugin_setting("mount_point", "secret")
       @path_prefix = fetch_plugin_setting("path_prefix", "inspec")
+      @priority = fetch_plugin_setting("priority", 60).to_i
 
       @vault = Vault::Client.new(
         address: fetch_vault_setting("vault_addr"),
         token: fetch_vault_setting("vault_token")
       )
     end
 
+    def default_priority
+      priority
```

Comment:
`default_priority` to me means that 60, not the currently set value. Seems like a bad name...

And if this is all it is, just use `alias`.

Reply:
`default_priority` is part of the [input plugin API](https://github.com/inspec/inspec/blob/master/docs/dev/plugins.md#default_priority) - it's an optional method you define. Some input providers may be able to set priority on a per-input basis (this one can't, or at least can't yet) so if not on a per-input basis, what's the default priority for an input?